Fix autogrow width issue (#4393)
ckeditor/ckeditor4: tests/plugins/autogrow/manual/4372.md

> @bender-tags: 4.15.1, bug, 7372

Author: I'm confused about one thing:
> We are at 4.15.1. version already. The upcoming release version for bugs is 4.15.2.

So I should tag it with a future version of the editor?

>7372 tag is invalid here. It should be the ticket number, so 4372 in our case.

:speak_no_evil: I mean it, so shame on me for this! :(

> @bender-tags: 4.15.1, bug, 7372

Reviewer: Yes, it should reflect the target release, not the current one, as we are not providing backward patches.

@bender-ui: collapsed

> @bender-ckeditor-plugins: wysiwygarea, autogrow

Reviewer: Let's add some minimal editor plugins, so the editor toolbar loads nicely.
```suggestion
@bender-ckeditor-plugins: wysiwygarea, autogrow, basicstyles, toolbar, resize
```

> 1. Focus inside editable window.
> 2. Add or remove extra lines to force editor resize.
> 3. Observe editor width.

Reviewer: ```suggestion
1. Focus the editor.
2. Press `enter` multiple times to force the editor to resize.
```

#### Expected result:

> The editor should have constant width. Only height is changing according to vertical amount of content.

Reviewer: ```suggestion
Editor width is not changing during auto-resize.
```

> #### Unexpected result:

Reviewer: ```suggestion
## Unexpected
```

> With each vertical resize editor is shrinking for about margins + paddings.

Reviewer: ```suggestion
With each vertical resize editor is shrinking for about 2 pixels.
```

Author: That was my first thought. Later it comes to my mind (I may be over-obsessed) if the default margin and padding will change, those `2px` will be invalid. Anyway, I apply suggestion :)
